src/test/java/com/exasol/dbbuilder/dialects/exasol/ExasolDatabaseObjectCreationAndDeletionIT.java

```diff
@@ -47,9 +49,12 @@ protected Connection getAdminConnection() throws SQLException {
         return container.createConnection("");
     }
 
-    @Override
-    protected DatabaseObjectFactory getDatabaseObjectFactory() throws SQLException {
-        return new ExasolObjectFactory(container.createConnection(""));
+    private static String getTableSysName(final DatabaseObject object) {
```

Comment:
as this method seems to be used only in `ExistsInDatabaseMatcher`,  I would suggest to move it there